CMakeLists.txt

```diff
@@ -281,6 +281,19 @@ if (NOT TECA_HAS_PARAVIEW)
 endif()
 set(TECA_HAS_VTK ${tmp} CACHE BOOL "VTK features")
 
+# configure for TELite
+set(tmp OFF)
+find_package(TELite QUIET)
+if (TELite_FOUND AND ((DEFINED TECA_HAS_TELITE AND TECA_HAS_TELITE) OR (NOT DEFINED TECA_HAS_TELITE)))
+    message(STATUS "TELite features -- enabled")
+    set(tmp ON)
+elseif (REQUIRE_TELITE)
+    message(FATAL_ERROR "TELite features -- required but not found. set TELite_DIR to enable.")
```

Comment:
I suggest that we set `REQUIRE_TELITE` to true by default; if a user does not want it, they can turn this flag off when building

Reply:
Where should I set it to true by default?

Reply:
I didn't find any other defaults set in any cmake file.

Reply:
not sure offhand then...though I know its possible

Reply:
Maybe in TECA_superbuild?

Reply:
that's definitely a place to set it too. But for TECA itself, here's what I'm suggesting: https://stackoverflow.com/a/53159688

Reply:
yes, that looks right

Reply:
Should we change anything in `docker/teca_cpu_debug/Dockerfile` and `setyp.py`?

Reply:
oh, yes - almost certainly: good catch!  does TE Lite need to be pre-compiled, or is it just the source directory that's needed?